packages/account/src/Components/forms/idv-form.tsx

```diff
@@ -112,7 +112,7 @@ const IDVForm = ({
 
     const onKeyUp = (e: { target: HTMLInputElement }, document_name: string) => {
         const { example_format } =
-            document_name === 'document_number' ? values.document_type : values.document_type.additional;
+            document_name === 'document_number' ? values?.document_type : values?.document_type.additional;
```

Comment:
There's a sonarcloud issue related to this line change: 
https://sonarcloud.io/project/issues?resolved=false&types=BUG&pullRequest=9623&id=binary-com_deriv-app&open=AYnU-iX_F9YsHZQwGbSw